.github/workflows/solutions_ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI for SAF templates
```

Comment:
Nitpick - I know we generally use `ci.yml` for our workflows, but I have been pushing to change that to something more descriptive. For example, in solutions, we renamed `ci.yml` to `build-release.yml`.

My stance here stems from the fact that since we are in the `workflows` folder, we already know this is related to CI/CD. I compare this to stuff like `var myVar = ...` etc.

Reply:
Very fair